en/docs/learn/role-based-permissions.md

```diff
@@ -34,7 +34,7 @@ role explained in
     enabled.
     
     !!! Tip 
-        You can change the default role name '**admin**' of the Administrative role as described in [here](../learn/configuring-roles-and-permissions/#updating-role-names)
+        You can change the default role name '**admin**' of the Administrative role as described in [here](../../learn/configuring-roles-and-permissions#updating-role-names)
```

Comment:
shall we change this to "as described here" instead of "in here"

Reply:
modified